scale/recipe/migrations/0037_recipetype_unique_together.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2019-04-04 12:34
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('recipe', '0036_remove_recipetype_version'),
+    ]
+
+    operations = [
+        migrations.AlterUniqueTogether(
+            name='recipetype',
+            unique_together=set([('name', 'revision_num')]),
```

Comment:
Why do we need this constraint?  RecipeType should be unique on name alone, not combined with revision_num.  Each entry will have the latest revision number and previous revision information is stored in the recipe type revision table.

Reply:
Removed file since we don't require another field to identify recipe types.